reference/6/PSDesiredStateConfiguration/PSDesiredStateConfiguration.md

```diff
@@ -11,20 +11,19 @@ Module Name:  PSDesiredStateConfiguration
 ---
 
 # PSDesiredStateConfiguration Module
+
 ## Description
-Windows PowerShell® is a task-based command-line shell and scripting language designed especially for system administration. This reference topic for the information technology (IT) professional provides assistance in utilizing the Windows PowerShell cmdlets to script and automate tasks.
+
+PowerShell is a task-based command-line shell and scripting language designed especially for system
+administration. This reference topic for the information technology (IT) professional provides
+assistance in utilizing the Windows PowerShell cmdlets to script and automate tasks.
```

Comment:
Missed a reference to 'Windows PowerShell'

Reply:
fixed